modules/API/external/external_api.js

```diff
@@ -366,6 +366,23 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         return iframe.contentWindow.document.getElementById('largeVideo');
     }
 
+    /**
+     * Getter for participant specific video element in Jitsi Meet.
+     * 
+     * @param {string} participantId - Participant id
+     * or "local" for local video.
+     * @returns {HTMLElement|undefined} - The requested video.
+     */
+    _getLargeVideo(participantId) {
+        const iframe = this.getIFrame();
+
```

Comment:
I would add some checks if the iframe, contentWindow, etc exist. Like the `_getLargeVideo`:
```
if (!iframe
                || !iframe.contentWindow
                || !iframe.contentWindow.document) {
            return;
        }
```

Reply:
Thanks, good feedback. Did as you suggested.